src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -75,6 +76,9 @@ function SidebarLinksData({isFocused, allReportActions, betas, chatReports, curr
         return reportIDs;
     }, [allReportActions, betas, chatReports, currentReportID, policies, priorityMode]);
 
+    const prevCurrentReportID = usePrevious(currentReportID);
+    const isActiveReport = useCallback((reportID) => prevCurrentReportID === reportID, [prevCurrentReportID]);
```

Comment:
Sorry, I know I proposed using `usePrevious`, but it's actually not a good idea here as it directly returns the value (ref.current). Plus its incorrect as it indeed returns the previous value.
This way the callback gets recreated for each currentReportID, which is something we want to avoid.

So instead you might want to consider the following code:

```suggestion
    const currentReportIDRef = useRef(currentReportID);
    currentReportIDRef.current = currentReportID;
    const isActiveReport = useCallback((reportID) => currentReportIDRef.current === reportID, []);
```